app/helpers/camaleon_cms/uploader_helper.rb

```diff
@@ -72,7 +72,7 @@ def upload_file(uploaded_io, settings = {})
       # save file
       key = File.join(settings[:folder], settings[:filename]).to_s.cama_fix_slash
       res = cama_uploader.add_file(settings[:uploaded_io], key, { same_name: settings[:same_name] })
-      {} if (settings[:temporal_time]).positive?
+      return {} if (settings[:temporal_time]).positive?
```

Comment:
This should not be an early return. There is a lot of code below this line that would get skipped if this change is merged.

[Looking back](https://github.com/owen2345/camaleon-cms/commit/0396bcf5731001dd3118844b3e24f264223dbb5d?diff=split&w=0#diff-bfb7e2a78971c0d660489c7833537465f3a547a00533a0326b4a7b44157d6930R58), this line was a placeholder with a TODO comment indicating that an operation performed in a thread should be moved to a background job. The hash literal appears to have been intended as an empty block where the code would go.

We can delete this line, or reimplement the function that was previously removed, but it should not be an early return.

Reply:
OK, I have re-implemented this function as the `CamaleonCmsUploader.delete_block` method whose block is defined in the `config/initializers/custom_initializers.rb` and can be overridden in the main app to wrap the block code in an async job.